Why data storage is shifting to the blockchain (#7164)
section-engineering-education/engineering-education: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md

## Why Data Storage is shifting to the blockchain

> Some years ago, the most embraced form of data storage was the traditional in-house data centers. But it was not too long before cloud storage started to replace them. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.

Reviewer: The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```suggestion
The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```

> In the same way, cloud storage is now facing competition from blockchain technology. Blockchain comes with the promise to solve data security challenges - an issue that cloud storage has found challenging to deal with exhaustively. With blockchain data storage, companies and individuals would enjoy the kind of data transparency not seen before. Thus, many companies are opting for this new technology with the hope that it could revolutionize data storage for the better.

Reviewer: ```suggestion
In the same way, cloud storage is now facing competition from blockchain technology. Blockchain promises to solve data security challenges - an issue that is a great threat to cloud storage. 

With blockchain data storage, companies and individuals will enjoy the desired data transparency. Thus, many companies are opting for this new technology with the hope of revolutionizing data storage.
```

> This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets leaders prepared of what to expect of blockchain as a data storage solution.

Reviewer: ```suggestion
This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets the reader to prepare for what to expect of blockchain as a data storage solution.
```

This storage method is different from the cloud, where data remains in one location and is easily accessible to an individual as a whole. The way blockchain stores its data in small chunks distributed throughout the network adds a security layer. It ensures that if an individual or entity breaks into the system, they can only access a fraction of the data rather than the entire file.

In cloud storage, data is mutable, while in blockchain, data is immutable. The cloud storage system is set up to let you change the already stored data. These modifications affect how another user perceives the data after the changes. Users cannot adjust the already existing data in the blockchain because the system bars them. You cannot delete or modify entries already made, which helps secure data from tampering.

Cloud storage relies on third parties to ensure data security. It provides services using the Software as a Service, Platform as a Service, and Infrastructure as a Service models. Blockchain chooses to guarantee data protection and safeguard its integrity without involving third parties.

> Besides, data in the cloud can exist as either public or private. However, in blockchain, data is always public. Blockchain embraces transparency, meaning that an entity cannot hide data from other users. It enables individuals on the blockchain to track how data moves from one point to another.

Reviewer: Hey, the best way to compare two technologies is using one characteristic and looking into it on both sides. For instance, if you take the mode of access, how does it occur in cloud storage and how different is it is from the blockchain.

Taking that approach makes your article smooth and add flow to your content.

### Costs implications of storing data on the blockchain

> Unsurprising, such a storage technology that threatens to 'overthrow' cloud storage and offer a highly secure system has its share of financial challenges. In the last few years, the costs for implementing and running a blockchain system have [risen](https://content-blockchain.org/newsarchive/2017/07/20/exploding-costs-of-storing-data-on-a-blockchain/) tremendously. One reason for this is the increasing acceptance of blockchain technology has also led to a rise in data transmission and storage. Besides, the high demand for blockchain services has also driven up cryptocurrency prices.

Reviewer: This paragraph sounds a bit off. Consider re-writing and splitting into two to convert a single idea per paragraph.

> There are specific factors that keep the blockchain functioning normally. In the case of bitcoin, the nodes and miners are responsible. For Ethereum, Gas is the fuel that maintains its routine running. Miners receive payment for their services, and in the case of Ethereum, one must pay for Gas.

Reviewer: How can you verify that the minors are responsible? 

`For Ethereum` ....... `and in the case of Ethereum`. Avoid unnecessary repetition.

How have the mentioned points contributed to the normal functioning of the blockchain?
